src/main/java/uk/ac/ebi/ampt2d/accession/serial/block/MonotonicRange.java

```diff
@@ -0,0 +1,161 @@
+/*
+ *
+ * Copyright 2017 EMBL - European Bioinformatics Institute
```

Comment:
Change copyrights to 2018 (this one should be easy!).

Reply:
It was committed on 2017 🙄